sdk/communication/Azure.Communication.Sms/README.md

```diff
@@ -48,25 +48,42 @@ SmsClient client = new SmsClient(new Uri(endpoint), tokenCredential);
 ## Examples
 ### Send a SMS Message
 To send a SMS message, call the `Send` or `SendAsync` function from the `SmsClient`.
-```C# Snippet:Azure_Communication_Sms_Tests_SendAsync
-SendSmsResponse result = await client.SendAsync(
-   from: new PhoneNumber("+18001230000"), // Phone number acquired on your Azure Communication resource
-   to: new PhoneNumber("+18005670000"),
-   message: "Hi");
-Console.WriteLine($"Sms id: {result.MessageId}");
+```C# Snippet:Azure_Communication_SmsClient_Send
+SmsSendResult result = smsClient.Send(
+    from: "+18001230000", // Phone number acquired on your Azure Communication resource
+    to: "+18005670000", // recipient's phone-number
```

Comment:
I do not think `to` and `message` need comment. For `from` we added the comment to make sure people know that they should can only use the phone numbers that they have acquired through the service, but the other two should be straightforward